admin-support-cli/tests/demotion-report-action.test.js

```diff
@@ -291,19 +293,16 @@ describe('Demotion report', () => {
         issueNumber: 1,
         duration: 1,
         ticket: '1234',
-        demotionDate: new Date(Date.parse('2021-03-19T11:05:50Z')),
-        promotionDate: new Date(Date.parse('2021-03-19T11:08:03Z')),
+        demotionDate: '2021-03-19T11:05:50Z',
+        promotionDate: '2021-03-19T11:08:03Z',
         targetOrg: 'test',
         reportPath: 'tests/fixtures'
       }
       replyGithubGetResponse('/orgs/test/audit-log', {
         include: 'all',
         per_page: 100,
-        phrase: `created:>=${mockParams.promotionDate} created:<=${mockParams.demotionDate} `
-      }, () => {
-        mockCallback()
-        return demotionAuditLog
-      })
+        phrase: `created:${mockParams.promotionDate}..${mockParams.demotionDate}`
+      }, demotionAuditLog)
```

Comment:
@stebje This was throwing a Javascript error that I didn't quite understand, and Copilot recommended this change. I'm thinking this might be related to the calling convention of `nock` vs. `fetch-mock`?